content/docs/Other guides/configuring-limits.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Configuring limits"
+linkTitle: "Configuring limits"
+weight: 4
+date: 2020-03-26
+description: >
+  Configure limits to set max number of concurrently running frames
+---
+
+This page describes how to configure limits which allow users to specify 
+the maximum number of concurrently running frames associated with that Limit.
+Limits are specified on the Job Layer and all Frames within that Layer are 
+considered to take 1 limit count. After the total limit count reaches the 
+configured max value, the dispatcher will stop any additional frames from 
+running until a frame with that limit has completed.
+
+## Configuring a new limit
+
+You can set a new limit by following these steps.
```

Comment:
Introduce lists with a colon, as follows:

You can set a new limit by following these steps: